photon-api/src/test/scala/com/linkedin/photon/ml/data/LocalDataSetTest.scala

```diff
@@ -51,6 +53,254 @@ class LocalDataSetTest {
     }
   }
 
+  @DataProvider
+  def dataForRandomEffectFeatureSelection(): Array[Array[Any]] ={
```

Comment:
I think it's the correct call to do one column at a time - it's not as clean, but it is much easier to handle the individual test cases and modify them.